oracle/OracleBillDownload.py

```diff
@@ -0,0 +1,81 @@
+import oci
+import os
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+import pytz
+import json
+
+# This script downloads all of the cost, usage, (or both) reports for a tenancy (specified in the config file).
+#
+# Pre-requisites: Create an IAM policy to endorse users in your tenancy to read cost reports from the OCI tenancy.
+#
+# Example policy:
+# define tenancy reporting as ocid1.tenancy.oc1..aaaaaaaaned4fkpkisbwjlr56u7cj63lf3wffbilvqknstgtvzub7vhqkggq
+# endorse group group_name to read objects in tenancy reporting
+#
+# Note - The only value you need to change is the group name. Do not change the OCID in the first statement.
+
+reporting_namespace = 'bling'
+
+# Download all usage and cost files. You can comment out based on the specific need:
+prefix_file = ""                     #  For cost and usage files
+# prefix_file = "reports/cost-csv"   #  For cost
+# prefix_file = "reports/usage-csv"  #  For usage
+
+# Update these values
+destintation_path = 'CBI'
+
+# Download all file or just past three months.
+env_download_all = os.environ.get("DOWNLOAD_ALL")
+download_all_files = bool(env_download_all)
```

Comment:
`bool(s)` returns `True` if the string `s` is not empty, else `False`. So `bool("False")` returns `True`, and `bool("")` returns False.